src/actions/AuthActions.js

```diff
@@ -31,8 +32,6 @@ const attemptFacebookLogin = async ({dispatch, navigate}) => {
             payload: {token, appReady: false}
         });
 
-        await AsyncStorage.setItem(tokens.USER_TOKEN, token);
```

Comment:
Now that `redux-persist` is handling everything about setting/removing things from `AsyncStorage` for us, we can remove any of our hard-coded references to it. Nice!!